setup.cfg

```diff
@@ -64,7 +64,7 @@ install_requires =
     pkginfo2 >= 30.0.0
     pip-requirements-parser >= 32.0.1
     requests >= 2.18.0
-    resolvelib >= 0.8.1
+    resolvelib @ git+https://github.com/sarugaku/resolvelib/@main
```

Comment:
This is quite a leap of faith! could you instead pin it to a commit?

Reply:
done

Reply:
updated now to version 1.0.0, which includes the fix